scenarios/kubernetes_e2e.py

```diff
@@ -345,9 +345,12 @@ def set_up_kops_aws(workspace, args, mode, cluster, runner_args):
     if args.aws_role_arn:
         profile = mode.add_aws_role(profile, args.aws_role_arn)
 
-    zones = args.kops_zones or random.choice(DEFAULT_AWS_ZONES)
-    regions = ','.join([zone[:-1] for zone in zones.split(',')])
-
+    zones = ""
+    regions = ""
+    if args.kops_multiple_zones is 'false'
```

Comment:
`if args.kops_multiple_zones:`

Reply:
it is a string ... not `False` ... does `not` work ... and I missed a colon ... I hate not being able to test this local